src/ch/ch_hostdev.c

```diff
@@ -39,3 +39,320 @@
 #define VIR_FROM_THIS VIR_FROM_CH
 
 VIR_LOG_INIT("ch.ch_hostdev");
+
+bool
+chHostdevHostSupportsPassthroughVFIO(void)
```

Comment:
Just reminds me we need to eventually get everything to be either virCH or just ch prefixed (I have been guessing function names incorrectly more often than not recently).